docs/notebooks/multiobjective_ehvi.pct.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+# # Multi-objective optimization with Expected HyperVolume Improvement Approach
+
+import math
+import gpflow
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from util.plotting import plot_bo_points, plot_function_2d, plot_mobo_history
+
+# +
+import trieste
+from trieste.acquisition.function import ExpectedHypervolumeImprovement
+from trieste.acquisition.rule import OBJECTIVE
+from trieste.data import Dataset
+from trieste.models import create_model
+from trieste.models.model_interfaces import ModelStack
+from trieste.space import Box
+from trieste.utils.multi_objectives import VLMOP2
+from trieste.utils.pareto import Pareto, get_reference_point
+
+np.random.seed(1793)
+tf.random.set_seed(1793)
+
+
+# -
+
+# ## Describe the problem
+#
+# In this tutorial, we provide multi-objective optimization example using the expected hypervolume improvement acquisition function.
+# We consider the VLMOP2 function --- a synthetic benchmark problem with two objectives. We start by defining the problem parameters.
+
+
+vlmop2 = VLMOP2().objective()
+observer = trieste.utils.objectives.mk_observer(vlmop2, OBJECTIVE)
+
+mins = [-2, -2]
+maxs = [2, 2]
+search_space = Box(mins, maxs)
+num_objective = 2
+
+# Let's randomly sample some initial data from the observer ...
+
+num_initial_points = 10
+initial_query_points = search_space.sample(num_initial_points)
+initial_data = observer(initial_query_points)
+
+# ... and visualise the data across the design space.
+
+_, ax = plot_function_2d(
+    vlmop2, mins, maxs, grid_density=100, contour=True, title=["Obj 1", "Obj 2"]
+)
+plot_bo_points(initial_query_points, ax=ax[0, 0], num_init=num_initial_points)
```

Comment:
Is it make sense of using a separate color bar for each subplot? As otherwise the color bar need to be properly scaled w.r.t `min` and `max` of all sub-plot, might be difficult to visualize in case the different objective function is not at the same scale (say one observation ranges from 1e10-1e100 the other 1-2)

Reply:
Yeah, two makes sense!